src/util-error.h

```diff
@@ -123,6 +123,7 @@ typedef enum {
     SC_ERR_DEBUG_LOG_GENERIC,
     SC_ERR_UNIFIED_LOG_GENERIC,
     SC_ERR_HTTP_LOG_GENERIC,
+    SC_ERR_FTP_LOG_GENERIC,
```

Comment:
only append to this enum to keep id's stable

Reply:
Will relocate constant.